src/httpServer/server.ts

```diff
@@ -304,7 +304,7 @@ export class DBOSHttpServer {
     const workflowStepsUrl = '/workflows/:workflow_id/steps';
     const workflowStepsHandler = async (koaCtxt: Koa.Context) => {
       const workflowId = (koaCtxt.params as { workflow_id: string }).workflow_id;
-      const steps = await dbosExec.listWorkflowSteps(workflowId);
+      const steps = await dbosExec.getWorkflowSteps(workflowId);
```

Comment:
Let's just do the stringification here the same way we do in Python (https://github.com/dbos-inc/dbos-transact-py/pull/272/)&mdash;Conductor code should remain completely isolated from the rest of the codebase.